Solutions/CohesitySecurity/Playbooks/Cohesity_CreateOrUpdate_ServiceNow_Incident/azuredeploy.json

```diff
@@ -0,0 +1,701 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Create or Update ServiceNow incident",
+        "description": "This playbook creates and updates the incident in the ServiceNow platform",
+        "prerequisites": "",
+        "postDeployment": [],
```

Comment:
Not seeing that all Entries related to metadata has been filled, Kindly fill them

Reply:
Alright. I copied the content from the readme-files, and now metadata is filled.
Out of my curiosity, can you please clarify why we need to keep prerequisites and deployment steps in both the json-file and the readme.md? We have readme-files for our user but who's metadata used?